### PR TITLE
feat: Feature/intmdb 1115 atlas basic tfmodule

### PR DESCRIPTION
**Pull request by govardhanpagidi**
feat:  Terraform module for atlas-basic quickstart

[INTMDB-1115](https://jira.mongodb.org/browse/INTMDB-1115)

To create atlas basic resources using terraform modules. 
This is just a replica of [AWS atlas basic Quickstart ](https://github.com/aws-quickstart/quickstart-mongodb-atlas/tree/8cd6e87ed36f467202432eed170a25ff10abf566)

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [x] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

File: modules/examples/atlas-basic/mian.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module "atlas-basic"  {
```
Comment (marcosuma) [inline modules/examples/atlas-basic/mian.tf]: please rename the file correctly to `main.tf`

**Comment by marcosuma** [comment]
Qq: did you try these examples? any chance you can put here confirmation they work?

**Comment by govardhanpagidi** [comment]
Example using terraform atlas-basic module

image

**Comment by govardhanpagidi** [comment]
This PR changes taken care with other [PR](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1494) which has atlas-basic and sagemaker changes

**Pull request closed without merging**